release-notes/release-notes58.md

```diff
@@ -25,7 +25,9 @@ keywords: releasenotes bugfixes
 
 The following new features were added for the {{site.productname}} 5.8 release.
 
-### Feature name
+### Add support for keys Home, End, PageUp and PageDown to escape inline elements
```

Comment:
```suggestion
### Improved keyboard navigation for inline elements
```

Reply:
### Improved keyboard navigation for the inline boundary feature